src/main/java/com/hms_networks/americas/sc/thingworx/TWConnectorMain.java

```diff
@@ -214,6 +219,17 @@ else if (queueDoubleDataRateEnabled && queueBehindMillis == 0) {
       } catch (Exception e) {
         Logger.LOG_CRITICAL("An error occurred while reading data from the historical log.");
         Logger.LOG_EXCEPTION(e);
+        // Chances are that trying the exact same read next time will result in the same exception,
+        // so advance the start time tracking
+        try {
+          HistoricalDataQueueManager.advanceTrackingStartTime();
+          Logger.LOG_WARN(
+              "Advancing historical start time, because an exception was thrown. This will"
+                  + " result in historical data loss.");
```

Comment:
I dont think this extra indent is needed. @alexjhawk ?

Reply:
This looks correct to me. I also just ran it through google java format in IntelliJ and it had no issues.

Reply:
Okay, nevermind about this then, Im just going crazy.